va/va.c

```diff
@@ -77,7 +77,7 @@ int va_parseConfig(char *env, char *env_value)
     while (fp && (fgets(oneline, 1024, fp) != NULL)) {
 	if (strlen(oneline) == 1)
 	    continue;
-        token = strtok_r(oneline, "=\n", &saveptr);
+    token = strtok_r(oneline, "=\n", &saveptr);
```

Comment:
I still see the wrong indentation when viewing the whole file in github.  the reason is that line 80 was indented by whitespace however line 78 was indented by a tab.